modules/ROOT/pages/values-and-types/property-structural-constructed.adoc

```diff
@@ -73,6 +73,7 @@ For more details, see xref::values-and-types/working-with-null.adoc[working with
 
 The table below shows the types and their syntactic synonyms.
 These types (and their synonyms) can be used in xref::syntax/expressions.adoc#type-predicate-expressions[type predicate expressions] and in xref::constraints/examples.adoc#constraints-examples-node-property-type[node] and xref::constraints/examples.adoc#constraints-examples-relationship-property-type[relationship] property type constraints.
+They are also output as a `STRING` value from the xref::functions/scalar.adoc#functions-valueType[valueType()] function.
```

Comment:
"output" is a little bit awkward as a verb. I think the below suggestion is accurate?

```suggestion
They are also returned as a `STRING` value when using the xref::functions/scalar.adoc#functions-valueType[valueType()] function.
```